lib/components/PreviewEmptyState.tsx

```diff
@@ -1,17 +1,16 @@
 import { Button } from "lib/components/ui/button"
 import { getRandomTipForUser } from "lib/utils/getRandomTipForUser"
 import { LightbulbIcon, PlayIcon } from "lucide-react"
-import { useMemo } from "react"
 
-const PreviewEmptyState = ({ onRunClicked }: { onRunClicked?: () => void }) => {
-  const tipHtml = useMemo(() => getRandomTipForUser(), [])
+const storedTipHtml: string = getRandomTipForUser()
```

Comment:
```suggestion
const tipHtml: string = getRandomTipForUser()
```

I think the previous variable name was good enough.